src/_DetailBase.js

```diff
@@ -567,9 +638,31 @@ const __class = declare('argos._DetailBase', [View], {
 
       if (!sectionStarted) {
         sectionStarted = true;
-        const section = domConstruct.toDom(this.sectionBeginTemplate.apply(layout, this) + this.sectionEndTemplate.apply(layout, this));
-        sectionNode = section.childNodes[1];
-        domConstruct.place(section, this.contentNode);
+        if (this.isTabbed) {
+          if (layout.name === 'QuickActionsSection') {
+            const section = domConstruct.toDom(this.sectionBeginTemplate.apply(layout, this) + this.sectionEndTemplate.apply(layout, this));
+            sectionNode = section;
+            domConstruct.place(section, this.quickActions);
+          } else {
+            const tab = domConstruct.toDom(this.tabListItemTemplate.apply(layout, this));
+            section = domConstruct.toDom(this.sectionBeginTemplate.apply(layout, this) + this.sectionEndTemplate.apply(layout, this));
+            sectionNode = section;
+            if (this.tabList.children.length === 0) {
+              // No children, so set the current tab to this tab and set the section to have a display of block
+              this.currentTab = tab;
+            } else {
+              section.style.display = 'none';
+            }
+            this.tabMapping.push(section);
+            domConstruct.place(tab, this.tabList);
+            this.checkTabOverflow(tab);
+            domConstruct.place(section, this.contentNode);
```

Comment:
Lines 659 and 664 are duplicates. Just move the functionality outside the if block so the section is only placed in one spot.